pages/posts/[id].vue

```diff
@@ -1,33 +1,41 @@
 <!-- eslint-disable vue/no-v-html -->
 <template>
   <div class="container mb-24">
-    <Breadcrumb>
-      <li>
-        <NuxtLinkLocale
-          class="fr-breadcrumb__link"
-          :external="true"
-          to="/"
-        >
-          {{ $t('Home') }}
-        </NuxtLinkLocale>
-      </li>
-      <li>
-        <NuxtLinkLocale
-          class="fr-breadcrumb__link"
-          to="/posts"
-        >
-          {{ $t('Posts') }}
-        </NuxtLinkLocale>
-      </li>
-      <li>
-        <a
-          class="fr-breadcrumb__link"
-          aria-current="page"
-        >
-          {{ post.name }}
-        </a>
-      </li>
-    </Breadcrumb>
+    <div class="flex items-center">
+      <Breadcrumb class="flex-1">
+        <li>
+          <NuxtLinkLocale
+            class="fr-breadcrumb__link"
+            :external="true"
+            to="/"
+          >
+            {{ $t('Home') }}
+          </NuxtLinkLocale>
+        </li>
+        <li>
+          <NuxtLinkLocale
+            class="fr-breadcrumb__link"
+            to="/posts"
+          >
+            {{ $t('Posts') }}
+          </NuxtLinkLocale>
+        </li>
+        <li>
+          <a
+            class="fr-breadcrumb__link"
+            aria-current="page"
+          >
+            {{ post.name }}
+          </a>
+        </li>
+      </Breadcrumb>
+      <div v-if="isAdmin(me)">
+        <EditButton
+          :id="(route.params.id as string)"
```

Comment:
This link to the edit page with the post slug in URL (it's working because the API return the post anyway but the aria-current=page is not because it's comparing slug with ID…) Not a big deal for now but maybe add a ticket on it.